common-entities/src/main/java/com/meeya/Building.java

```diff
@@ -1,11 +1,20 @@
 package com.meeya;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
 import java.util.List;
 
 /**
  * @author anurag.y
  * @since 08/10/18.
  */
+@Getter
```

Comment:
* Use `@Data` annotation to skip all these imports.
* Add `@Builder` annotation to provide flexibility to build object in a safe way.

**Note:** Apply for each file.